evaluation-tasks/config/coughvid.py

```diff
@@ -0,0 +1,33 @@
+"""
+Configuration for the coughvid task
+"""
+
```

Comment:
Some of these could probably be moved to a global / luigi config file (similar to the s3 config stuff that I pulled out)

Reply:
https://github.com/neuralaudio/hear2021-eval-kit/issues/10